torch/testing/_internal/distributed/rpc/dist_autograd_test.py

```diff
@@ -2712,17 +2796,26 @@ def test_dist_autograd_sync_streams(self):
         remote_compute = rpc.remote(dst, TensorPipeCudaDistAutogradTest.MyRemoteCompute)
         local_compute = TensorPipeCudaDistAutogradTest.MyLocalCompute(remote_compute)
         for _ in range(10):
-            input = torch.rand([1000, 10000], device=self.rank, requires_grad=True)
+            if sparse:
+                input = build_sparse_tensor(requires_grad=True, device=self.rank)
```

Comment:
can be done in followup PR: shall we try larger scale scale tensors with randomly generated indices and values? Looks like, the dense part of this test tries 1000 X 10000, but the sparse part only tries 3 elements?

Reply:
That makes sense. It might make sense in the future to update tests that use random values instead to use a set of combinations. For example, the set can be generated using the boundary testing technique.